### ISSUE TITLE
Catalyst App Dark Appearance Button Iems Not Visible

### ISSUE THREAD

**Issue by zewkini** [guoyingtao/Mantis#420]
In the Normal and Transformation Views you can't see the button Items and any associated text when running the app in Dark Mode:

![Screenshot 2024-10-18 at 5 39 40 PM](https://github.com/user-attachments/assets/7da84c20-3405-4897-a249-2f7d3ae2ec7e)

**Comment by guoyingtao** [comment]
@zewkini 
What version of Mantis are you using? Can you update to the latest version and test again? I tested it on my MacBook Pro and it looked fine.

<img width="465" alt="image" src="https://github.com/user-attachments/assets/c7160e0b-4efd-4711-afd4-e35bea59a5ba">

**Comment by zewkini** [comment]
@guoyingtao 

Using v2.23.0. I ran it on my 2019 Mac Pro (Sonoma). What I ended up trying was changing `backgroundColor = .white` to `backgroundColor = .systemBackground`  line 128 in `CropToolbar`. That seems to work.

Dark Appearance seems to run differently depending on whether or not you've selected Optimize for Mac or Scaled to Match iPad.

![Screenshot 2024-10-19 at 4 19 03 PM](https://github.com/user-attachments/assets/ef175ac2-2256-43cd-96c6-2d50ddaab8d6)

**Comment by guoyingtao** [comment]
@zewkini 

What Xcode version are you using? I am using Xcode 16.0 and I found currently the line 128 in CropToolbar won't be called when debugging for Mac Catalyst. 
I remembered it was called before...

**Comment by zewkini** [comment]
@guoyingtao 

I'm using Xcode 16 as well. I'm using the package as written so (in CropToolbar):

`if #available(macCatalyst 14.0, iOS 14.0, *)` at line 126 always executes and if the `UIDevice.current.userInterfaceIdiom == .mac ` resolves to true (which it does when running on my mac) then `backgroundColor = .white` is set, as it should be.

I'm not very familiar with if #available statement but to me it looks like it's working as intended?

If I comment out the `backgroundColor = .white` then the toolbar appears black, which is set in CropToolbarConfig

```
/**
     The color settings are not for Mac Catalyst (Optimize Interface for Mac) for now
     I haven't figured out a correct way to set button title color for this scenario
     */
    public var backgroundColor: UIColor = .black
    public var foregroundColor: UIColor = .white
```

Maybe I'm missing something. My knowledge of XCode and Swift is pretty basic but it looks like the program execution is working the way it is coded.

**Comment by zewkini** [comment]
@guoyingtao 

After further review, I looked over my project settings and discovered that I had Mac as one of the devices in the Targeted Device Family. That was what was causing the problem. I deleted that device and it works fine now. I apologize for not discovering that sooner.
